Sources/protoc-gen-swift/MessageStorageClassGenerator.swift

```diff
@@ -49,7 +49,11 @@ class MessageStorageClassGenerator {
       // readonly usage never causes the allocation.
       p.print("""
 
-          static let defaultInstance = _StorageClass()
+          #if swift(>=5.10)
+            static nonisolated(unsafe) let defaultInstance = _StorageClass()
```

Comment:
Can we generate a code comment here detailing why we use `nonisolated(unsafe)` and why that usage is in fact safe.

Reply:
Sure, what would you like to it to say?

Reply:
Something like this
```
// This property is used as the initial default value for new instances of the type.
// The type itself is protecting the reference to its storage via CoW semantics. This will force a copy to be made
// of this reference when the first mutation occurs; hence, it is safe to mark this as `nonisolated(unsafe)`.
```

Reply:
Done.